main.py

```diff
@@ -1,11 +1,58 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player in players:
+        race_name = players[player]["race"]["name"]
+        race_description = players[player]["race"]["description"]
+        new_race = Race.objects.filter(name=race_name).exists()
+        if not new_race:
+            Race.objects.create(name=race_name, description=race_description)
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create